### PR TITLE
feat: add functionality to filter by item type in most viewed items c…

### LINKED ISSUE #71: add a translation function for the word "Count" in the app's bar charts
Each of `ActionsByTimeOfDayChart.js`, `ActionsByDayChart.js`, and `ItemsByAccessedCount.js` have this Recharts component:

`<Bar dataKey="count" name="Count" fill="#8884d8" />`

The `name` needs to be changed to `{t('Count')}` so that this text is translated

### LINKED ISSUE #72: add functionality to filter by item type for 'most viewed items' chart

### PR DESCRIPTION
**Pull request by hagoptaminian**
…hart

also add translation wrapper to word 'count' in bar charts

closes #72, closes #71

### CODE REVIEW

**Comment by hagoptaminian** [comment]
@juancarlosfarah the chart looks as follows:

Screen Shot 2021-03-31 at 2 47 19 PM

note that I didn't use the custom component in the Select dropdown to display 'x, y, **and N others selected**'as we did with the user filter, so the dropdown looks like this:

Screen Shot 2021-03-30 at 11 04 30 PM

as opposed to this:

Screen Shot 2021-03-30 at 11 04 47 PM

...since there are typically less than a handful of item types and it doesn't make sense to display _...and M others selected_ for such a small number

File: src/components/custom/ActionsByTimeOfDayCustomTooltip.js [hunk 1]
```diff
@@ -54,7 +56,7 @@ const CustomTooltip = ({ active, payload, label }) => {
       <div className={classes.customTooltipDiv}>
         <p>{`${label} (${generateAddedTooltipText(label)})`}</p>
         <p className={classes.customTooltipCount}>
-          {`Count : ${payload[0].value}`}
+          {`${t('Count')} : ${payload[0].value}`}
```
Comment (juancarlosfarah) [inline src/components/custom/ActionsByTimeOfDayCustomTooltip.js:59]: I think you can bring the `:` inside the `t` function. In some languages (French) you have a space before the colon and in others (English) you don't.

File: src/components/custom/ItemsByAccessedCountCustomTooltip.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  if (active) {
+    return (
+      <div className={classes.customTooltipDiv}>
+        <p className={classes.heading}>{`${label} (${label})`}</p>
+        <p>{`${t('Category')}: ${payload[0].payload.category}`}</p>
+        <p className={classes.customTooltipCount}>
+          {`${t('Count')}: ${payload[0].payload.count}`}
```
Comment (juancarlosfarah) [inline src/components/custom/ItemsByAccessedCountCustomTooltip.js:35]: Make `:` consistent with other tooltip.

File: src/components/space/functionality/ItemsSelect.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        closeMenuOnSelect={false}
+        hideSelectedOptions={false}
+        allowSelectAll
+        getOptionLabel={(option) => option.name}
+        value={selectedItemTypes}
+        onChange={(selected) => {
+          if (
+            selected !== null &&
+            selected.length > 0 &&
+            selected[selected.length - 1].value === allOption.value
```
Comment (juancarlosfarah) [inline src/components/space/functionality/ItemsSelect.js:37]: What does this mean? Maybe add a comment to be explicit.

**Review by juancarlosfarah — APPROVED** [review]
Looks good! Just a minor question / request to add a comment. But otherwise 💯 !

File: src/components/custom/ActionsByTimeOfDayCustomTooltip.js (hunk 1 above)
Comment (hagoptaminian) [reply src/components/custom/ActionsByTimeOfDayCustomTooltip.js:59]: Aaah interesting. I did not know this. Recharts in its default Tooltips adds a space, which I didn't like. I'll do this the correct way (for English) by bringing the colon into the `t()`.

File: src/components/space/functionality/ItemsSelect.js (hunk 3 above)
Comment (hagoptaminian) [reply src/components/space/functionality/ItemsSelect.js:37]: 👍

**Pull request merged**